documentation/breaking-changes/upcoming-breaking-changes.md

```diff
@@ -25,4 +25,103 @@
     https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes/breaking-change-template.md
 -->
 
-# Upcoming Breaking Changes
\ No newline at end of file
+# Upcoming Breaking Changes
+
+## Release 3.0.0 - February 2017
+
+    The following cmdlets were affected this release:
+
+    **Get-AzureRmMetricDefinition**
+    - The metric definitions are now retrieved from MDM. In this case, the records returned by this cmdlet are different even though the call remains the same.
+
+    ```powershell
+    # Old
+  
+    $s1 = Get-AzureRmMetricDefinition -res $resourceIdMetric -det
+    write-host "Values: $s1"
+    $s11 = $s1[0]
+    write-host "First value: $s11"
+    $metricName = $s11.Name
+    
+    $s2 = Get-AzureRmMetricDefinition -res $resourceIdMetric -met $metricName -det
+    $s21 = $s2[0]
+    write-host "Availabilities: $s21.MetricAvailabilities"
+    
+    write-host "Unit: $s21.Unit"
+    write-host "ResourceId: $s21.ResourceId"
+    write-host "Name: $s21.Name"
+    write-host "PrimaryAggregationType: $s21.PrimaryAggregationType"
+    write-host "Properties: $s21.Properties"
+    
+    $s22 = $s21.MetricAvailabilities[0]
+    write-host "BlobLocation: $s22.BlobLocation"
+    write-host "Location: $s22.Location"
+    write-host "Retention: $s22.Retention"
+    write-host "Timegrain: $s22.TimeGrain"
+
+    # New
+  
+    $s1 = Get-AzureRmMetricDefinition -res $resourceIdMetric -det
+    write-host "Values: $s1"
+    $s11 = $s1[0]
+    write-host "First value: $s11"
+    $metricName = $s11.Name.Value
+    
+    $s2 = Get-AzureRmMetricDefinition -res $resourceIdMetric -met $metricName -det
+    $s21 = $s2[0]
+    write-host "Availabilities: $s21.MetricAvailabilities"
+    
+    write-host "Unit: $s21.Unit"
+    write-host "ResourceId: $s21.ResourceId"
+    write-host "Name: $s21.Name.Value"
+    write-host "PrimaryAggregationType: $s21.PrimaryAggregationType"
+    
+    $s22 = $s21.MetricAvailabilities[0]
+    write-host "Retention: $s22.Retention"
+    write-host "Timegrain: $s22.TimeGrain"
+    
+    ```
+  
+  **Get-AzureRmMetric**
+    - The metrics are now retrieved from MDM. The call and the returned records are different now.
+
+    ```powershell
+    # Old
+    
+    # ResourceId and timeGrain are mandatory and positional
+    $s1 = Get-AzureRmMetric $resourceIdMetric $timeGrain -det
+    
+    write-host "Values: $s1"
+    
+    $s11 = $s1[0]
+    
+    write-host "Dimension name: $s11.DimensionName"
+    write-host "Dimension value: $s11.DimensionValue"
+    write-host "End time: $s11.EndTime"
+    write-host "Start time: $s11.StartTime"
+    write-host "ResourceId: $s11.ResourceId"
+    write-host "TimeGrain: $s11.TimeGrain"
+    write-host "Unit: $s11.Unit"
+    write-host "Metric values: $s11.MetricValues"
+
+    # New
+  
+    # This calls are valid. ResourceId remains mandatory and positional
```

Comment:
I made some changes to this file. My goal here is to describe all the changes including the outputs, even if the calls remains the same as is the case with the Metric Definitions